testdata/steward.toml

```diff
@@ -0,0 +1,14 @@
+[sgx]
+signer = ["c8dc9fe36caaeef871e6512c481092754c57c2ea999f128282ccb563d1602774"]
+hash = ["e106565074be5ef3897711472617a0a000bae5c577f69a42202e3f76a07980f3"]
+features = ["Debug", "ProvisioningKey", "EIntKey", "KSS"]
+enclave_security_version = 0
+enclave_product_id = 0
+
+[snp]
+signer = ["5b2181f5e2294fa0709d22b3f85d9d88b287b897c6b7289004802b53bbf09bc50f5469f98a6d6718d5f9c918d3d3c16f"]
+id_key_digest = ["966a25a22ee44283aa51bfb3682c990fd9e0a7457c5f60f4ac4eb5c41715478c4b206b0e01dc11aae8628f5aa29e0560"]
+hash = ["6ff9ac4c61adc4cd2d86264230afc386358a5b41221dd236de92a3b45cb8a590bf719388994711ab9fe2b192bebc18a2"]
```

Comment:
Having `id_key_digest` and just plain `hash` fields is not a good UX. (along a multitude other `.*digest.*` fields)
This configuration should be consistent and preferrably self-documenting.
I think `hash` field should be renamed.

What about `memory_digest`? `runtime_digest`? @haraldh what do you think is the most appropriate here?

IMO just calling it `digest` and filing an issue to revisit the config option naming is good enough as well, but we really need a ticket to track this, and it should be addressed before we deploy this Steward version to production.

@puiterwijk from deployer's perpective, what naming scheme would make the most sense? Asking since you're going to be the one actually using and maintaining these configs

Reply:
Agreed. I think I had it called something else before, but it was suggested that the name wasn't similar enough to the field being validated. Just need to decide on something and stick with it.

Reply:
or `keep_measurement`